runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -191,6 +213,51 @@ public void onGameStateChanged(GameStateChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onMenuOptionClicked(MenuOptionClicked event)
+	{
+		// Warn for auto-retaliate?
```

Comment:
Is this a TODO comment? It should either be implemented if you meant to implement it, or removed otherwise.